src/assets/translations/messages-en.json

```diff
@@ -108,6 +108,9 @@
       }
     }
   },
+  "staking": {
+    "staking": "Staking"
```

Comment:
Shouldn't this be `Staking & Vaults` ?

Reply:
Good question - Given yearn vaults has its own section and this will only be used for cosmos validators, it didn't make sense to add vaults. But design might be changing this entire section to condense all of the `earn` sections into one, so it might be changing soon.

Reply:
Makes sense 👍🏽 in this case, `Staking` is a better wording. Independently from design choices, the `StakingOpportunities` is very similar to the existing `EarnOpportunities` so I added some mention in https://gist.github.com/gomesalexandre/cfa9fa943bcdbbde36ca6e82a7889d01#opportunities - it would be nice to have one shared generic component when we start breaking things down.